### ISSUE TITLE
footline problem with 3.31

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#273]
Originally reported by: **Anonymous**

----------------------------------------

Hello,

We're experiencing a problem with beamer 3.31. It worked with 3.27.
Aim: logo in bottom left corner, page number in bottom right corner.
In 3.31 however, the logo becomes centered on the page and both "Some text" and page number are no longer visible.

Regards,
Stefan

````
\defbeamertemplate*{footline}{my theme dark}
{%
  \pgftext[left,bottom,at=\pgfpoint{0.02\paperwidth}{0.055\paperwidth}]{\pgfuseimage{mylogo}}%
  \pgftext[left,bottom,at=\pgfpoint{0.0275\paperwidth}{0.037\paperwidth}]{%
   \usebeamerfont{frame number in foot}%
   \usebeamercolor[cantgrey]{frame number in foot}Some%
 }%
  \pgftext[left,bottom,at=\pgfpoint{0.0280\paperwidth}{0.02\paperwidth}]{%
   \usebeamerfont{frame number in foot}%
   \usebeamercolor[cantgrey]{frame number in foot}text%
 }%
 \pgftext[right,bottom,at=\pgfpoint{0.97\paperwidth}{0.02\paperwidth}]{%
    \usebeamerfont{frame number in foot}%
    \usebeamercolor[cantgrey]{frame number in foot}\insertframenumber{}/\inserttotalframenumber%
  }%
}
````

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/273

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

This is I think fixed in the development version, which will be released as v3.32 once I have one additional patch landed (hopefully this week).  See https://bitbucket.org/rivanvx/beamer/commits/5cf6c5555a45d5a80bebaf6bd630960e94a71e95 for where the issue is: the block of code this is in is a bit long for a 'quick patch' (sorry).

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

I'm assuming this is fixed
